syntax/phpblock.php

```diff
@@ -22,7 +22,7 @@ public function getPType()
     /** @inheritDoc */
     public function getSort()
     {
-        return 190;
+        return 180;
```

Comment:
While I agree with this change (makes the plugin override the default in pre "Jack Jackrum" DokuWiki), shouldn't this be a separate PR?

Also why 180 and not 189 as for the generic case (`syntax_plugin_htmlok_htmlphpok`)? As a matter of fact, why override the generic case at all?